Libplanet/Net/Swarm.cs

```diff
@@ -1364,6 +1364,11 @@ CancellationToken cancellationToken
                     $"(tip: {blockChain.Tip?.Hash})"
                 );
 
+                if (receivedBlockCount + hashCount > totalBlockCount)
```

Comment:
I'd prefer `Math.max()` in these cases. (but it isn't a strong opinion.)

```
totalBlockCount = Math.Max(totalBlockCount, receivedBlockCount + hashCount);
```

Reply:
I changed it as your suggestion.